packages/nx-adsp/src/generators/angular-app/files/src/polyfills.ts__tmpl__

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
curious about this file... is the polyfills not included in the base nrwl angular generator, and/or are we modifying it in some way?

Reply:
yeah, if we don't add 


 import '@angular/localize/init';


to the bottom of the file, we get an error - I think the angular package in ui-components has the same file line at the bottom